aonvif/client.py

```diff
@@ -298,9 +336,12 @@ async def update_xaddrs(self):
 
         # Get XAddr of services on the device
         self._xaddrs = {}
-        capabilities = await devicemgmt.GetCapabilities({'Category': 'All'})
-        for name in capabilities:
-            capability = capabilities[name]
+
+        if not self._capabilities:
+            self._capabilities = await devicemgmt.GetCapabilities({'Category': 'All'})
+
+        for name in self._capabilities:
```

Comment:
`for name, capability in self._capabilities.items()`?

Reply:
`AttributeError: Capabilities instance has no attribute 'items'`

Reply:
Go it, thanks!